src/commands/coffeechat/coffeeMatch.ts

```diff
@@ -1,41 +1,38 @@
 import { Message, User } from 'discord.js';
-import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { Command, CommandOptions } from '@sapphire/framework';
+import { ApplyOptions } from '@sapphire/decorators';
+import { container } from '@sapphire/pieces';
 import { writeHistoricMatches, getMatch } from '../../components/coffeechat';
-import { AdminCommand } from '../../utils/commands';
-import { logError } from '../../components/logger';
 
-class CoffeeMatchCommand extends AdminCommand {
-  constructor(client: CommandoClient) {
-    super(client, {
-      name: 'coffee-match',
-      aliases: ['coffeematch'],
-      group: 'coffeechats',
-      memberName: 'match',
-      description: 'Matches current coffee chat roles',
-      examples: [`${client.commandPrefix}coffeematch`]
-    });
-  }
-
-  async onRun(message: CommandoMessage): Promise<Message> {
+@ApplyOptions<CommandOptions>({
+  aliases: ['coffee-match'],
+  description: 'Matches current coffee chat roles.',
+  detailedDescription: `**Examples:**\n\`${container.client.options.defaultPrefix}coffeematch\``,
+  requiredUserPermissions: ['ADMINISTRATOR']
+})
+export class CoffeeMatchCommand extends Command {
```

Comment:
\n maybe

Reply:
Hmmm, none of the other commands have `\n`. Should I still have `\n`? If so, in a separate PR, I can add `\n` to the other commands as well.

Reply:
I would have expected a new line. IG leave it as is.

Reply:
hmm cmd names should start with coffeechat tho